curriculum/challenges/english/25-front-end-development/workshop-city-skyline/5d822fd413a79914d39e9917.md

```diff
@@ -22,6 +22,13 @@ Add a `repeating-linear-gradient` to `.fb1c` below the one that's there; use you
 
 # --hints--
 
+You should make sure the background value for .fb1c exists and contains no syntax errors.
```

Comment:
```suggestion
You should make sure the background value for `.fb1c` exists and contains no syntax errors.
```

We do not want values like `.fb1c` to be translated by Crowdin, thusly we mark those values with backticks.